README.md

```diff
@@ -136,6 +136,32 @@ val schemaId = schemaManager.register(subject, schemaString)
 // and more, check SchemaManager's methods
 ```
 
+### De-serialisation Error Handling
+There are 2 ways ABRiS handles de-serialisation errors:
+
+#### FailFast (Default)
+Given no provided de-serialisation handler, a failure will result in a spark exception being thrown 
+and with the error being outputted. This is the default procedure.
+
+#### SpecificRecordHandler
+The second option requires providing a default record that will be outputted in the event of a failure.
+This should be used as a flag to be deleted outside ABRiS that should mean the spark job will not stop. 
+Beware however, a null or empty record will also result in an error so a record with a different input should be chosen.
+
+This can be provided as such:
+```scala
+val abrisConfig = AbrisConfig
+  .fromConfluentAvro
+  .downloadReaderSchemaByLatestVersion
+  .andTopicNameStrategy("topic123")
+  .usingSchemaRegistry(registryConfig)
+  .withSchemaConverter("custom")
+  .withExceptionHandler(new SpecificRecordExceptionHandler(providedDefaultRecord))
+```
+
+This is only for confluent-based configuration, not for standard avro.
```

Comment:
Did you verify that `withExceptionHandler` is available only for the `fromConfluentAvro` configuration and not for others?

Reply:
Yeah it only works/appears for Confluentavro